### PR TITLE
add task column to judge queue view

### PR DESCRIPTION
**Pull request by joeyyang**
Resolves https://github.com/department-of-veterans-affairs/caseflow/issues/8652
/cc @allyceh 

### Description
Adds the task column to judge queue view (with the task name in `_.startCase`)

screenshot 2019-01-25 10 16 18

### CODE REVIEW

File: client/COPY.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "CASE_LIST_TABLE_APPEAL_ISSUE_COUNT_COLUMN_TITLE": "Issues",
   "CASE_LIST_TABLE_APPEAL_DOCUMENT_COUNT_COLUMN_TITLE": "Veteran Documents",
   "CASE_LIST_TABLE_TASK_ACTION_COLUMN_TITLE": "Action",
+  "CASE_LIST_TABLE_TASK_LABEL_COLUMN_TITLE": "Task(s)",
```
Comment (lowellrex) [inline client/COPY.json]: Can we re-use the existing `CASE_LIST_TABLE_TASKS_COLUMN_TITLE` copy here instead?

File: client/app/queue/components/TaskTable.jsx [hunk 2]
```diff
@@ -131,6 +132,13 @@ export class TaskTableUnconnected extends React.PureComponent<Props> {
     } : null;
   }
 
+  caseTaskLabelColumn = () => {
+    return this.props.includeTaskLabel ? {
+      header: COPY.CASE_LIST_TABLE_TASK_LABEL_COLUMN_TITLE,
+      valueFunction: (task: TaskWithAppeal) => _.startCase(task.label)
+    } : null;
+  }
+
```
Comment (lowellrex) [inline client/app/queue/components/TaskTable.jsx]: This seems very similar to the existing [`caseTaskColumn` task table function](https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/app/queue/components/TaskTable.jsx#L126). Can we re-use that? Perhaps by title-casing the label in [in the `actionNameOfTask` function](https://github.com/department-of-veterans-affairs/caseflow/blob/36e9250177ddce645b96fa59fe919128f1449e6d/client/app/queue/components/TaskTable.jsx#L124) if the task label does not exist in the set of `CO_LOCATED_ADMIN_ACTIONS` we map the label into?

File: client/COPY.json (hunk 1 above)
Comment (joeyyang) [reply client/COPY.json]: done!

File: client/app/queue/components/TaskTable.jsx (hunk 2 above)
Comment (joeyyang) [reply client/app/queue/components/TaskTable.jsx]: done!

**Pull request merged**